dhall-docs/src/Dhall/Docs/Html.hs

```diff
@@ -66,6 +78,19 @@ exprToHtml expr = renderTree prettyTree
             Builtin -> "builtin"
             Operator -> "operator"
 
+    typeLayout :: Pretty.Doc ann -> Pretty.SimpleDocStream ann
+    typeLayout = Pretty.removeTrailingWhitespace . Pretty.layoutSmart opts
+      where
+        -- this is done so the type of a dhall file fits in a single line
```

Comment:
Laying out all types on a single line is not really ideal. These lines could get _very_ long. I'm fine with improving the layout at a later stage though.

Reply:
I'll add it to my task list to not forget this.